main.tf

```diff
@@ -0,0 +1,15 @@
+module "db_instance" {
```

Comment:
If there is only one resource, there is no need to use a module to implement it.